l10n_be_national_number/data/res_partner_id_category.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+  <record id="l10n_be_national_number_category" model="res.partner.id_category">
+    <field name="code">l10n_be_national_number</field>
+    <field name="name">Belgium National Number</field>
```

Comment:
shouldn’t this be “Belgian national registration number” (translated to “Numéro de registre national belge”)? or something like “Belgium: national registration number” or ”National registration number (Belgium)”? https://sma-help.bosa.belgium.be/en/faq/where-can-i-find-my-national-registration-number